packages/flutter/lib/src/cupertino/segment_control.dart

```diff
@@ -0,0 +1,191 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/painting.dart';
+
+class SegmentControlItem {
+  final String title;
+  final Widget content;
+
+  SegmentControlItem(this.title, this.content);
+}
+
+class SegmentControl extends StatefulWidget {
+  final List<SegmentControlItem> tabs;
+  final int activeTabIndex;
+
+  SegmentControl(this.tabs, {this.activeTabIndex = 0})
```

Comment:
Let tabs/segments be a named argument for consistency with the rest of the codebase like `children` on rows, `items` on CupertinoTabBar etc.

Use `:` for default value.